code/extensions/FileSubsites.php

```diff
@@ -120,22 +138,24 @@ public function canEdit($member = null)
     {
         // Check the CMS_ACCESS_SecurityAdmin privileges on the subsite that owns this group
         $subsiteID = Session::get('SubsiteID');
```

Comment:
Best substitution is
```
if (Injector::inst()->has(HTTPRequest::class)) {
	$request = Injector::inst()->get(HTTPRequest::class);
	$session = $request->getSession();
} else {
	throw new Exception("some error");
}
```

In 4.x we have been moving this kind of state management out of direct session getters into middleware; Let me know if you'd like me to write one up for you. Otherwise, you can use the above code as a placeholder to get this PR over the line.

Reply:
Ah, why not just use `Subsite::currentSubsiteID()`? Legacy code confuses me lol.